### ISSUE TITLE
At 100% "BlazorWebAssemblyStandalone" template name is truncating and not visible clearly :A11y_ASP.NET_BlazorWebAssemblyStandalone_Usability

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspnetcore#56296]
### GitHub Tags:
#A11yUsable; #A11yTCS; #A11ySev4; #BM_ASP.NET_Web_Jun2024; #.NETCore; #DesktopWeb; #Win11; #FTP; #ChromiumEdge;

### Environment Details:
App name: ASP.NET
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Download the "BlazorWebAssemblyStandalone" zip file.
2. Open command prompt and enter cd "<UNZIPPED_FOLDER_PATH>" command and hit ENTER key.
3. Enter "dotnet serve" command next and hit ENTER key.
4. TAB to "http://localhost:54293/" link and press CTRL + ENTER key.
5. Observe that At 100% "BlazorWebAssemblyStandalone" template name is truncating and not visible clearly

### Actual Result:
At 100% "BlazorWebAssemblyStandalone" template name is truncating and not visible clearly.

### Expected Result:
At 100% "BlazorWebAssemblyStandalone" template name should be visible clearly.

### User Impact:
Low vision users will not be able to see the controls clearly and will not get information about it.

### Attachment:

![56296_A11y_ASP NET_BlazorWebAssemblyStandalone_Usability](https://github.com/dotnet/aspnetcore/assets/90901519/fea41898-9c31-4d43-87e9-075a2da4ab49)

**Comment by mkArtakMSFT** [comment]
This is due to the fact that the project name used during creation was long. We won't be fixing this given that this was due to the unusual input as well as the Sev4 nature of the bug.

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;
